content/docs/rfcs/34/README.md

```diff
@@ -86,6 +89,8 @@ message PeerExchangeRPC {
 The `enr` field contains a Waku ENR as specified in [31/WAKU2-ENR](/spec/31/).
 
 Requesters send a `PeerExchangeQuery` to a peer.
+When `cluster_id` is set and `shards` are specified, responder SHOULD provide peers belonging to cluster_id/shards pair.
+When no `shards` are specified, responder CAN provide mixed list of peers from any shard and non-shard speific peers. 
```

Comment:
```suggestion
When no `shards` are specified, responders SHOULD provide a mixed list of peers from any shard and non-shard specific peers. 
```
`CAN` is not a recognised RFC keyword: https://www.ietf.org/rfc/rfc2119.txt and in such cases you should use `MAY`. In this case, though, I would say that the relationship is `SHOULD`.